front-end/src/app/shared/components/glossary/glossary.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Really love this solution!  My gut reaction was that there should be a display name and it should be different from the key, but since it's used as the search term as well, this is just really simple and to-the-point until we have more requirements on this